ivy/functional/ivy/random.py

```diff
@@ -35,7 +35,7 @@ def random_uniform(
         If the given shape is, e.g ``(m, n, k)``, then ``m * n * k`` samples are drawn.
         If size is ``None`` (Default), a single value is returned.
     device
-        device on which to create the array 'cuda:0', 'cuda:1', 'cpu' etc.
+        device on which to create the array 'gpu:0', 'gpu:1', 'cpu' etc.
```

Comment:
You can change this part back to `'cuda:0', 'cuda:1', 'cpu' etc. `

Also in a new line, please include: ` (Default value = None)` just like in Line 226.